docs/src/5.1.MigratingToVersion0.2.x.mdx

```diff
@@ -0,0 +1,91 @@
+```js
+// New type
+type ClickedObject = {
+  object: Object,
+  instanceIndex?: ?number,
+};
+
+export type ReglClickInfo = {
+  ray: Ray,
+  objects: Array<ClickedObject>,
+};
+
+export type MouseHandler = (MouseEvent, ReglClickInfo) => void;
+```
+
+Usage:
+
+```js
+// Old
+<Worldview
+  onClick={(event, { ray, objectId }) => {
+    setSelectedObjectId(objectId);
+  }}>
+  ...
+</Worldview>
+// New
+<Worldview
+  onClick={(event, { ray, objects }) => {
+    setSelectedObjects(objects.map(({ object, instanceIndex }) => object));
+  }}
+  enableStackedObjectEvents>
+  ...
+</Worldview>
+```
+
+## Hitmap changes
+
+How do we detect which object was interacted with? We use a process called picking, in which we generate an off-screen canvas, or "hitmap", to paint our objects into, assign each object a unique color, and then paint each object into this hitmap. We then detect which object was clicked on by testing which color was underneath the cursor.
+
+For stacked objects, we run this hitmap rendering multiple times. We note which object was picked, then exclude it from the next rendering cycle, until we stop hitting objects.
+
+Previously, the assignment of these colors (also referred to as `hitmapIds`) was passed into each Command. Now there is an API for drawing these colors in a more scalable way.
+
+This means that we no longer support passing the `getObjectFromHitmapId`, `getHitmapId`, `getHitmapProps`, or `hitmapProps` props into a Command instance.
+
+Instead we have a single function, `getChildrenForHitmap`. This function takes as arguments the children passed into the Command, another function that helps assign colors for the hitmap, and a list of objects that we want to exclude from rendering (used for stacked object hitmaps). It returns new children that we use for rendering to the hitmap.
+
+Similarly to the previous API, there are a few default `getChildrenForHitmap` functions available to you to use.
+
+```js
+// Old
+<Cubes getHitmapId={getHitmapId} getObjectFromHitmapId={getObjectFromHitmapId}>{cubes}</Cubes>
+
+// New
+<Cubes getChildrenForHitmap={createInstancedGetChildrenForHitmap(1)}>{cubes}</Cubes>
+```
+
+See [MouseEvents](#/docs/api/mouse-events) for further info on the new hitmapId API.
+
+## Command syntax changes
+
+Because the Command has been simplified, there wasn't any need for a `makeCommand` wrapper any longer. The `makeCommand` function has been removed. Simply wrap the `Command` in another React component that passes it's props through, while specifying the `getChildrenForHitmap` function and the `reglCommand`. It's recommended that the `getChildrenForHitmap` function be overrideable and the `reglCommand` not be overrideable through props - see [MouseEvents](#/docs/api/mouse-events) for more on this.
```

Comment:
```suggestion
Because the Command has been simplified, there wasn't any need for a `makeCommand` wrapper any longer. The `makeCommand` function has been removed. Simply wrap the `Command` in another React component that passes its props through, while specifying the `getChildrenForHitmap` function and the `reglCommand`. It's recommended that the `getChildrenForHitmap` function be overrideable and the `reglCommand` not be overrideable through props — see [MouseEvents](#/docs/api/mouse-events) for more on this.
```